buildLogAnalyzer.py

```diff
@@ -380,6 +380,7 @@ def makeHTMLSummaryPage(self):
 
     def makeHTMLLogFile(self, pkg):
         """docstring for makeHTMLFile"""
+        linePartsUrl = re.compile(r"\s*(src(/[^:]+):(\d+)):.*")
```

Comment:
@iarspider , this will not work for all release cycles. For old releases we have full path.